lib/css.js

```diff
@@ -1,31 +1,29 @@
 'use strict';
 
-var merge = require('lodash').merge;
-var cssnano = require('gulp-cssnano');
-var cssnext = require('postcss-cssnext');
-var gulpif = require('gulp-if');
-var importer = require('postcss-import');
-var postcss = require('gulp-postcss');
+import {merge} from 'lodash';
+import cssnano from 'gulp-cssnano';
+import cssnext from 'postcss-cssnext';
+import gulpif from 'gulp-if';
+import importer from 'postcss-import';
+import postcss from 'gulp-postcss';
 
-var defaults = {
+const defaults = {
   dest: './dist',
   optimize: false,
   src: './src/main.css'
 };
 
-module.exports = function (gulp, options) {
+module.exports = (gulp, options) => {
   var opts = merge({}, defaults, options);
   var dest = opts.dest;
   var optimize = opts.optimize;
   var src = opts.src;
 
-  gulp.task('css', function () {
-    return gulp.src(src)
-      .pipe(postcss([
-        importer(),
-        cssnext()
-      ]))
-      .pipe(gulpif(optimize, cssnano()))
-      .pipe(gulp.dest(dest));
-  });
+  gulp.task('css', () => gulp.src(src)
+                          .pipe(postcss([
+                            importer(),
+                            cssnext()
+                          ]))
+                          .pipe(gulpif(optimize, cssnano()))
+                          .pipe(gulp.dest(dest)));
```

Comment:
@erikjung: I wasn't sure here about indentation. Since technically the `return` was a one-liner, I removed the `{ }` and the actual explicit `return`. Is this too much? All tests passed when I ran `npm test`.